src/together/legacy/files.py

```diff
@@ -53,9 +53,11 @@ def upload(
 
         client = together.Together(api_key=api_key)
 
-        response = client.files.upload(file=file).model_dump()
+        # disabling the check, because it was run previously
```

Comment:
Are the implementations in legacy used, and if not is this required?

Reply:
To the best of my understanding, it is the legacy interface, but `files.upload` calls the new method in `together/resources/files.py` which I updated. Hence, it's affected by the change to the `upload` method